config/core/test-configs.yaml

```diff
@@ -1318,6 +1324,27 @@ device_types:
     <<: *qemu_x86_64
     params: *uefi_i386
 
+  r8a7743-iwg20d-q7:
+    mach: renesas
+    class: arm-dtb
+    boot_method: uboot
+    filters:
+      - passlist: {defconfig: ['defconfig']}
+
+  r8a7745-iwg22d-sodimm:
+    mach: renesas
+    class: arm-dtb
+    boot_method: uboot
+    filters:
+      - passlist: {defconfig: ['defconfig']}
+
+  r8a77470-iwg23s-sbc:
```

Comment:
This one has been offline most of the time, and has never run a KernelCI job:
https://lava.ciplatform.org/scheduler/device_type/r8a77470-iwg23s-sbc
Is the dtb upstream?